skymp5-server/cpp/server_guest_lib/PieScript.cpp

```diff
@@ -3,14 +3,22 @@
 #include "FormDesc.h"
 #include "MpActor.h"
 #include <random>
+#include <sstream>
 #include <vector>
 
 std::mt19937 g_rng{ std::random_device{}() };
 
 int GenerateRandomNumber(int leftBound, int rightBound)
 {
-  std::uniform_int_distribution<> distr(leftBound, rightBound);
-  return distr(g_rng);
+  if (leftBound <= rightBound) {
+    std::uniform_int_distribution<> distr(leftBound, rightBound);
+    return distr(g_rng);
+  } else {
+    std::stringstream ss;
+    ss << "GenerateRandomNumber() cannot generate number in range: ("
+       << leftBound << ", " << rightBound << ")\n";
+    throw std::runtime_error(ss.str());
```

Comment:
Better to use `throw std::runtime_error(fmt::format("GenerateRandomNumber() cannot generate number in range: ({}, {})", leftBound, rightBound));`